processor/src/tasks/processGerbers/index.ts

```diff
@@ -1,7 +1,7 @@
 import globule from 'globule'
 import Jszip from 'jszip'
+import fs from 'node:fs/promises'
```

Comment:
Shouldn't these get deleted too? 

https://github.com/kitspace/kitspace-v2/blob/89a686c48bb4595f69bfd728a7e1bfd9bd739371/processor/src/tasks/processGerbers/index.ts#LL11C3-L11C3

Reply:
I think each job should delete its `outputFiles` when it's done.

Reply:
Oops, I accidentally removed the whole deletion. I've amended the last commit.